docs/source/schema/directives.md

```diff
@@ -105,18 +105,19 @@ If you want to define a custom schema directive to _transform_ your executable s
 
 ### In subgraphs
 
-> ⚠️ **Important considerations**
->
-> Before you use directives in a federated graph, make sure to consider the following:
->
-> - Custom directives are _not_ included in your graph's composed supergraph schema. The composition process strips all subgraph directives. Only a given subgraph is aware of its own directives.
-> - Because directives are specific to individual subgraphs, it's valid for different subgraphs to define the _same_ directive with _different_ logic. Composition does not detect or warn about such inconsistencies.
-> - If multiple subgraphs can resolve a particular field, each subgraph should almost always apply the exact same set of custom directives (with the exact same accompanying logic) to that field. Otherwise, the behavior of that field might vary depending on _which_ subgraph resolves it.
+Before you use custom directives in a federated graph, make sure to consider the following:
+- If multiple subgraphs can resolve a particular field, each subgraph should almost always apply the exact same set of custom directives (with the exact same definition) to that field. Otherwise, the behavior of that field might vary depending on _which_ subgraph resolves it.
+- Because directives are specific to individual subgraphs, it's technically valid for different subgraphs to define the _same_ directive with _different_ logic. As stated in the previous point, if a custom directive is used in multiple subgraphs to resolve a particular field, you should define the same directive with the same logic across subgraphs. _Composition does not detect or warn about such inconsistencies._
+- The composition process treats [executable (client-side)](/federation/federated-types/composition/#executable-directives) and [type system (server-side) directives](/federation/federated-types/composition/#type-system-directives) differently:
+  - An executable directive is composed into the supergraph schema if:
+      - All subgraphs define the directive identically 
+      - The directive is not included in any [`@composeDirective`](/federation/federated-types/federated-directives/#composedirective) directives
+  - Type system directives are not composed into the supergraph schema, but they can provide information to the router via the [`@composeDirective`](/federation/federated-types/federated-directives/#composedirective) directive.
```

Comment:
We may clarify here that is is not "to the Router" but instead included in the supergraph schema, which is then used by the Router and coprocessors

Reply:
Ah, so `@composeDirective` _does_ compose type system directives onto the supergraph schema? I was taking the verbiage from [here](https://www.apollographql.com/docs/federation/federated-types/composition/#type-system-directives).

I can update both places if that's incorrect.

Reply:
@Meschreiber That language is incorrect. `@composeDirective` DOES make type-system directives available to the supergraph schema. The API schema (the one clients see) does not expose any directive info though, but it never does this for any type system directives, with the exception of `@deprecated`


![Screenshot 2023-08-21 at 12 24 29 PM](https://github.com/apollographql/apollo-server/assets/2446877/bd440611-50f5-45bb-aff2-e761d306242f)

![Screenshot 2023-08-21 at 12 24 40 PM](https://github.com/apollographql/apollo-server/assets/2446877/ff375647-4902-44fc-b8bf-78f46a8d22db)

Reply:
Thank you!